product_maintenance_equipment_link/models/product.py

```diff
@@ -0,0 +1,9 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    maintenance_equipment_ids = fields.Many2many("maintenance.equipment")
```

Comment:
Add a string attribute, because default translation shows it in singular, not plural:

![imagen](https://github.com/user-attachments/assets/e41094e6-9f7c-4194-93d1-38b285e83ae4)